security/compliance_operator/compliance-operator-crd.adoc

```diff
@@ -0,0 +1,509 @@
+
+[id="compliance-suite-object_{context}"]
+=== ComplianceSuite object
+The `ComplianceSuite` object helps you to keep a track on the state of the scans. It contains the raw settings to create scans and the overall result.
+
+For `Node` type scans, you should map the scan to the `MachineConfigPool` as it contains the remediations for any issues. If you specify a label, ensure it directly applies to a pool.
+
+.Example `ComplianceSuite` object
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ComplianceSuite
+metadata:
+  name: <name of the scan>
+spec:
+  autoApplyRemediations: false <1>
+  schedule: "0 1 * * *" <2>
+  scans: <3>
+    - name: workers-scan
+      scanType: Node
+      profile: xccdf_org.ssgproject.content_profile_moderate
+      content: ssg-rhcos4-ds.xml
+      contentImage: quay.io/complianceascode/ocp4:latest
+      rule: "xccdf_org.ssgproject.content_rule_no_netrc_files"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+status:
+  Phase: DONE <4>
+  Result: NON-COMPLIANT <5>
+  scanStatuses:
+  - name: workers-scan
+    phase: DONE
+    result: NON-COMPLIANT
+----
+<1> Set to `true` to enable auto remediations. Set to `false` to disable auto remediations.
+<2> Specify how often the scan should be run in cron format.
+<3> Specify a list of scan specifications to run in the cluster.
+<4> Indicates the progress of the scans.
+<5> Indicates the overall verdict of the suite.
+
+The suite in the background creates the `ComplianceScan` object based on the `scans` parameter.
+You can programmatically fetch the `ComplianceSuites` events. To get the events for the suite, run the following command:
+[source,terminal]
+----
+$ oc get events --field-selector involvedObject.kind=ComplianceSuite,involvedObject.name=<name of the suite>
+----
+
+[IMPORTANT]
+====
+You might induce errors when you manually define the `ComplianceSuite` as it contains the XCCDF attributes.
+====
+
+[id="advance-compliance-scan-object_{context}"]
+=== Advanced ComplianceScan Object
+The Compliance Operator includes options for advanced users for the purpose of debugging or integration with existing tooling. While it is recommended that not to create a `ComplianceScan` object directly, instead you can manage it by a `ComplianceSuite` object.
+
+.Example Advanced `ComplianceScan` object
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ComplianceScan
+metadata:
+  name: <name of the scan>
+spec:
+  scanType: Node <1>
+  profile: xccdf_org.ssgproject.content_profile_moderate <2>
+  content: ssg-ocp4-ds.xml
+  contentImage: quay.io/complianceascode/ocp4:latest <3>
+  rule: "xccdf_org.ssgproject.content_rule_no_netrc_files" <4>
+  nodeSelector: <5>
+    node-role.kubernetes.io/worker: ""
+status:
+  phase: DONE <6>
+  result: NON-COMPLIANT <7>
+----
+
+<1> Specify either `Node` or `Platform`. Node profiles scan the cluster nodes and platform profiles scan the Kubernetes platform.
```

Comment:
Use "Specifies" for these annotations.

Reply:
@lpettyjo  Jakub mentioned these are set by the user.